### PR TITLE
License updates to be consistent on new services

### PR DESCRIPTION
**Pull request by cartersocha**
Adding consistent licensing across new services. 

@puckpuck could you point me where to add comments in erlang/elixir? I'm lost 😆 @xoscar similar question for the frontend

@mtwo does this blurb work? I just updated the google text

### CODE REVIEW

**Comment by cartersocha** [comment]
I may be missing some files. Tried to cover most of what we added or replaced with a new language

**Review by austinlparker — APPROVED** [review]
lgtm -- 1 comment, I don't know if we need license blurbs at the top of the docker files/compose files? Is that consistent with other SIGs?

File: docker-compose.yml [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Copyright 2022, OpenTelemetry Authors
```
Comment (reyang) [inline docker-compose.yml:1]: Remove the year, it's hard to maintain and adds little if no value.

https://github.com/open-telemetry/opentelemetry-java/blob/19da306e9aefa35b79314d4876e319a743566b95/api/all/src/jmh/java/io/opentelemetry/api/baggage/propagation/W3CBaggagePropagatorBenchmark.java#L2

**Review by reyang — DISMISSED** [review]
https://github.com/open-telemetry/opentelemetry-demo/pull/486/files#r999692129

**Comment by cartersocha** [comment]
> lgtm -- 1 comment, I don't know if we need license blurbs at the top of the docker files/compose files? Is that consistent with other SIGs?

I noticed Google doing that but removed the blurb for now. I kept it on the docker compose file since that was a big net new addition

**Comment by cartersocha** [comment]
towards: https://github.com/open-telemetry/opentelemetry-demo/issues/220

File: docker-compose.yml [hunk 2]
```diff
@@ -1,3 +1,17 @@
+# Copyright OpenTelemetry Authors
```
Comment (reyang) [inline docker-compose.yml:1]: Add "The"

https://github.com/open-telemetry/community/blob/f9740a5ac0c6bab5cbbb3f477134d7e59eb12071/CONTRIBUTING.md#copyright-notices

File: src/currencyservice/src/server.cpp [hunk 3]
```diff
@@ -1,5 +1,16 @@
-// Copyright The OpenTelemetry Authors
-// SPDX-License-Identifier: Apache-2.0
+ // Copyright The OpenTelemetry Authors
+ //
```
Comment (reyang) [inline src/currencyservice/src/server.cpp:2]: Fix the indentation here:

```suggestion
//
```

**Comment by cartersocha** [comment]
@reyang I think I got all the spaces removed

**Review by reyang — APPROVED** [review]
LGTM。

**Comment by cartersocha** [comment]
@tsloughter any suggestions on where I add the licensing blurb to the feature flag service? I've just been doing it in the main file of all the new additions

**Comment by cartersocha** [comment]
Ping on this too @xoscar

**Comment by tsloughter** [comment]
@cartersocha usually each `.erl` and `.ex` file will have the blurb at the top

**Comment by cartersocha** [comment]
> @cartersocha usually each `.erl` and `.ex` file will have the blurb at the top

Okay thanks. To minimize changes I just add a single blurb to the ffs_service file. I think I did it correctly

**Comment by tsloughter** [comment]
yup, looks good

File: src/currencyservice/src/server.cpp [hunk 4]
```diff
@@ -120,7 +131,7 @@ class CurrencyService final : public hipstershop::CurrencyService::Service
 
     span->AddEvent("Currencies fetched, response sent back");
     span->SetStatus(StatusCode::kOk);
-    // Make sure to end your spans!
+   // Make sure to end your spans!
```
Comment (mviitane) [inline src/currencyservice/src/server.cpp:134]: I don't see a reason for this indentation change. I think it was better before (aligned with the code).

This comment is valid for all similar changes below.

Comment (cartersocha) [reply src/currencyservice/src/server.cpp]: Good catch. I had a space bug I fixed in bulk and this changed too. Resolved

**Pull request merged**